notebooks/ICESat-2_MODIS_Arctic_Sea_Ice/Customize and Access Data.ipynb

```diff
@@ -40,6 +40,17 @@
     "import requests\n",
```

Comment:
<p>the following lines can be omitted, since these packages are in the <code>tutorial_helper_functions</code> already (but keep <code>import math</code> in there!) :</p><p> </p><pre class="ql-syntax" spellcheck="false">import socket
import zipfile
import io
import os
import shutil
import pprint
import re
import time
from statistics import mean
from requests.auth import HTTPBasicAuth 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>